share/completions/git.fish

```diff
@@ -850,16 +850,16 @@ complete -c git -n '__fish_git_using_command diff show range-diff' -s B -l break
 complete -c git -n '__fish_git_using_command diff show range-diff' -s b -l ignore-space-change -d 'Ignore changes in amount of whitespace'
 complete -c git -n '__fish_git_using_command diff show range-diff' -s C -l find-copies -d 'Detect copies as well as renames'
 complete -c git -n '__fish_git_using_command diff show range-diff' -s D -l irreversible-delete -d 'Omit the preimage for deletes'
-complete -c git -n '__fish_git_using_command diff show range-diff' -s G -d 'Look for differences whose patch text contains added/removed lines that match <regex>'
+complete -c git -n '__fish_git_using_command diff show range-diff' -s G -d 'Look for differences where <regex> matches the patch text''s added/removed lines'
```

Comment:
Doubling up single quotes just closes and then reopens the quotes. (I think it works like this in rc?)

You'll have to escape it with `\\` or use double-quotes for the outside.

Reply:
Also just "the patch's added/removed lines" should work? The "patch text" part is weird and I think unnecessary.